config/initializers/active_admin.rb

```diff
@@ -108,7 +108,7 @@
   #
   # This setting changes the method which Active Admin calls
   # (within the application controller) to return the currently logged in user.
-  # config.current_user_method = :current_admin_user
+  config.current_user_method = :current_user
```

Comment:
I don't see this defined outside of app/controllers/api/v1; you get it for free from Devise, correct?

Reply:
I think so, yes, I also removed it from api/v1 as it wasn't used there either